format/spec.md

```diff
@@ -1680,6 +1680,23 @@ Row-level delete changes:
     * These position delete files must be merged into the DV for a data file when one is created
     * Position delete files that contain deletes for more than one data file need to be kept in table metadata until all deletes are replaced by DVs
 
+Row lineage changes:
+
+* Writers must set the table's `next-row-id` and use the existing `next-row-id` as the `first-row-id` to create a new snapshot
```

Comment:
```suggestion
* Writers must set the table's `next-row-id` and use the existing `next-row-id` as the `first-row-id` when creating new snapshots
```

Reply:
I considered this language, but I think it's more clear to use a singular snapshot. I don't want to imply that you would use `next-row-id` for multiple snapshots.

Reply:
You're right. I committed this change. I think I'm just being pedantic.